src/Select.js

```diff
@@ -879,9 +880,17 @@ const Select = React.createClass({
 		}
 
 		for (var i = 0; i < options.length; i++) {
-			if (!options[i].disabled) return i;
+			// if there is no selected option just use first value in options
+			if (!options[i].disabled && !focusedOption) {
+				return i;
+			} else if (!options[i].disabled && options[i].value === focusedOption.value) {
+				return i;
+			} else if (!options[i].disabled && !selected) {
+				// get first option thats not disabled
+				selected = i;
```

Comment:
should be:
`return selected = options[i];`

This way it will be consistent with the other return statements in the for loop.